<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditServeurWidget</class>
 <widget class="QWidget" name="AuditServeurWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audit serveur mono-client</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Num√©ro de port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Informations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLancement">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancement Serveur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPort">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditServeurWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
